cfgov/core/utils.py

```diff
@@ -35,3 +35,15 @@ def extract_answers_from_request(request):
     answers = [(param.split('_')[1], value) for param, value in
                request.POST.items() if param.startswith('questionid')]
     return answers
+
+
+def slice_list(l, n):
```

Comment:
@richaagarwal nice catch; I'd introduced this when experimenting with multithreaded. I've removed this in 51c98e6.